book/src/bitwise.md

```diff
@@ -0,0 +1,40 @@
+# Bitwise Access in ST

```

Comment:
I dont relly remember where the term "bitwise access" origins from (wasnt this basically used by dominik this way? - turning the german phrase "bitweiser Zugriff" into this english term?)
nevertheless ... the norm does not mention this term and when you google bitwise access you dont find this concept 
... the closest you find are bitwise functions (AND, OR, XOR).

The norm calls this: "Directly represented variables" and I think we should also use this term

Reply:
Codesys call it "Bit Access" https://help.codesys.com/api-content/2/codesys/3.5.13.0/en/_cds_operands_variables_accessing_bits/
I will call the page direct_variables and the title will be " Direct (Bit) Access on Variables